src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSinkUnitTest.java

```diff
@@ -52,4 +62,18 @@ public void readsSinkTest(String inputBam, String outputFile) throws IOException
         Assert.assertEquals(rddParallelReads.count(), rddParallelReads2.count());
     }
 
-}
\ No newline at end of file
+    @Test(dataProvider = "loadReadsADAM", groups = "spark")
+    public void readsSinkADAMTest(String inputBam, String outputFile) throws IOException {
+        new File(outputFile).deleteOnExit();
+        JavaSparkContext ctx = SparkContextFactory.getTestSparkContext();
+
+        ReadsSparkSource readSource = new ReadsSparkSource(ctx);
+        JavaRDD<GATKRead> rddParallelReads = readSource.getParallelReads(inputBam);
+        SAMFileHeader header = ReadsSparkSource.getHeader(ctx, inputBam);
+
+        ReadsSparkSink.writeReads(ctx, outputFile, rddParallelReads, header, ReadsWriteFormat.ADAM);
+
+        JavaRDD<GATKRead> rddParallelReads2 = readSource.getADAMReads(outputFile, header);
+        Assert.assertEquals(rddParallelReads.count(), rddParallelReads2.count());
```

Comment:
I don't quite follow, the issue is closed. Is a problem not yet fixed at head with Adam? Fixed, but not released?
If the fix isn't coming to us soon, just add a comment here saying it's an issue with Adam and file a Hellbender ticket to track it.